cms/static/js/views/course_info_handout.js

```diff
@@ -53,7 +53,12 @@ define(['js/views/baseview', 'codemirror', 'common/js/components/views/feedback_
                 $('#handout_error').removeClass('is-shown');
                 $('.save-button').removeClass('is-disabled').attr('aria-disabled', false);
                 if ($('.CodeMirror-lines').find('.cm-error').length == 0) {
-                    this.model.set('data', this.$codeMirror.getValue());
+                    if (this.$codeMirror.getValue() == '') {
+                        this.model.set('data', '<ol></ol>');
```

Comment:
What if we only init the `data-to-be-saved` in the `if-else` condition and after we do somthing like
`this.model.set('data', 'data-to-be-set');`